test/manual/definitions/otel/debians/debian11-otel.json

```diff
@@ -0,0 +1,34 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+
+    "resources": [{
+        "id": "host1",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.nano",
+        "ami_name": "debian-11-amd64-2021????-*",
+        "user_name": "admin"
+    }],
+
+    "instrumentations": {
```

Comment:
For all deploy configs under `tests/manual`, we do not want to add instrumentations, since that defeats the purpose of having uninstrumented instances for manual testing. Please delete these blocks

Reply:
I have deleted the manual test definitions as they are same as infra resources.